### PR TITLE
Proof of Concept: Radio CSS states

### PR DESCRIPTION
**Pull request by rabelloo**
Only opening so that we get a preview deploy.
Will close later.

### CODE REVIEW

**Comment by josokinas** [comment]
Great PoC, applied almost everything to https://github.com/onfido/castor/pull/79 now.

File: packages/react/src/components/radio/radio.react.tsx [hunk 1]
```diff
@@ -1,52 +1,26 @@
-import { c, classy, RadioProps as BaseProps, m } from '@onfido/castor';
-import React, { useState } from 'react';
+import { c, classy, m, RadioProps as BaseProps } from '@onfido/castor';
```
[the remark this answers is not in the document]
Comment (rabelloo) [reply packages/react/src/components/radio/radio.react.tsx:1]: Not a plugin, it's a VSCode config, `settings.json`
```
"editor.codeActionsOnSave": {
  "source.organizeImports": true
},
```

Or you can manually press `shift + option + O`

[the remark this answers is not in the document]
Comment (rabelloo) [reply packages/react/src/components/radio/radio.react.tsx:1]: Can't find the issue we had for implementing that as a lint rule or something 🤔

[the remark this answers is not in the document]
Comment (josokinas) [reply packages/react/src/components/radio/radio.react.tsx:1]: I don't think there's issue created, feel free to do so!

**Pull request closed without merging**